smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/model/ShapeExt.kt

```diff
@@ -54,7 +54,7 @@ fun Shape.changeNameSuffix(fromTo: Pair<String, String>): String {
  * If is member shape returns target, otherwise returns self.
  * @param model for loading the target shape
  */
-internal fun Shape.targetOrSelf(model: Model) = when (this) {
+fun Shape.targetOrSelf(model: Model): Shape = when (this) {
     is MemberShape -> model.expectShape(this.target)
     else -> this
 }
```

Comment:
**Comment**: Oh nice, didn't know this function existed. I'm pretty sure I've duplicated logic this elsewhere.